src/templates/community-champion-profile.js

```diff
@@ -257,6 +273,230 @@ const Bio = styled(P)`
+                  Icon: TwitterIcon,
+              },
+          ]
+        : [];
+    return (
+        <div>
+            <LinksTitle collapse>Social</LinksTitle>
+            <SocialLinksContainer>
+                {links.map(
+                    ({ type, url, Icon }) =>
+                        url && (
+                            <SocialLink
+                                key={type}
+                                icon={
+                                    <Icon
+                                        height={size.default}
+                                        width={size.default}
+                                    />
+                                }
+                                type={type}
+                                link={url}
+                            />
+                        )
+                )}
+            </SocialLinksContainer>
+        </div>
+    );
+};
+
+const SocialLinkItemContainer = styled(Link)`
+    align-items: center;
+    column-gap: ${size.xsmall};
+    display: grid;
+    grid-template-columns: ${size.default} auto;
+    text-decoration: none;
+    path {
+        fill: ${({ theme }) => theme.colorMap.devWhite};
+    }
+    &:hover {
+        path {
+            fill: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const SocialLinksContainer = styled('div')`
+    column-gap: ${SOCIAL_LINKS_GRID_COLUMN_GAP};
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-columns: repeat(2, max-content);
+    grid-template-rows: repeat(3, auto);
+    row-gap: ${size.xsmall};
+    @media ${screenSize.upToSmallDesktop} {
+        column-gap: ${size.mediumLarge};
+        grid-auto-flow: row;
+        grid-template-columns: repeat(3, max-content);
+        grid-template-rows: repeat(2, auto);
+    }
+    @media ${screenSize.upToSmall} {
+        grid-template-columns: repeat(
+            auto-fill,
+            minmax(${SOCIAL_LINK_MAX_WIDTH}, 1fr)
+        );
+        grid-template-rows: none;
+    }
+`;
+
+const SocialLinkText = styled(P2)`
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.small};
+        line-height: ${lineHeight.small};
+    }
+`;
+
+const SocialLink = ({ icon, link, type }) => (
+    <SocialLinkItemContainer href={link}>
+        {icon}
+        <SocialLinkText collapse>{type}</SocialLinkText>
+    </SocialLinkItemContainer>
+);
+
+const CertificationsContainer = styled('div')`
+    column-gap: ${CERTIFICATIONS_CONTAINER_COLUMN_GAP};
+    display: flex;
+    flex-wrap: wrap;
+    row-gap: ${size.default};
+    @media ${screenSize.upToSmallDesktop} {
+        column-gap: ${size.large};
+        margin-bottom: ${size.xlarge};
+    }
+`;
+
+const CertificationsSection = ({ certifications }) => (
+    <div>
+        <Title>MongoDB Certifications</Title>
+        <CertificationsContainer>
+            {certifications.isDBAAssociateCertified && (
+                <Link href="https://mongodb.com">
```

Comment:
fyi - this is just a placeholder link! i didn't realize we needed to link to the certificates, so i will need to update some fields on strapi on monday!